conf/github-contributions.nginx.conf

```diff
@@ -26,4 +35,14 @@ server {
         alias /github-archive/timeline/;
         autoindex on;
     }
+
+    location ~ /.well-known {
+        allow all;
+    }
```

Comment:
This makes it so instead of routing `/.well-known` via the `/` block (with `proxy_pass` in it), it just goes directly to `$doc_root/.well-known`, and without this line you'll get 403s for some reason.